apiserver/remoterelations/remoterelations_test.go

```diff
@@ -177,15 +211,16 @@ func (s *remoteRelationsSuite) TestWatchRemoteApplicationRelationRemovedRace(c *
 		RemoteRelationsWatcherId: "1",
 		// The relation is not found, but it was never reported
 		// to us, so it should not be reported in "Removed".
-		Changes: &params.RemoteRelationsChange{},
+		Change: &params.RemoteRelationsChange{},
 	}})
 	s.st.CheckCalls(c, []testing.StubCall{
 		{"WatchRemoteApplicationRelations", []interface{}{"db2"}},
 		{"KeyRelation", []interface{}{"db2:db django:db"}},
 	})
 }
 
-func (s *remoteRelationsSuite) TestWatchRemoteApplicationRelationUnitRemoved(c *gc.C) {
+// TODO(wallyworld) - underlying code not currently used
```

Comment:
This whole section of code will be refactored - the code is essentially 12 months old and several branches are required to get things in shape. All this code is hidden behind a feature flag.